benchmarks/shelley3pools/create-addresses-generic.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
why is this file "generic"? how does it differ from the normal version?

Reply:
The tx generator works by iterating over a set of addresses. The way we can have 3 nodes all submit transactions is by having 3 different sets of addresses. Thinking about it, we could have just one set of addresses and the nodes could submit transactions to the same set... I was keeping them all separate as it was easier to debug at the time.